esmvaltool/config-user_MISTRAL.yml

```diff
@@ -0,0 +1,26 @@
+# ESMValTool User's configuration file for DKRZ-MISTRAL
```

Comment:
Was this accidentally committed?

Reply:
Yes, that was an accident. I removed the file now.